supervise-daemon-guide.md

```diff
@@ -101,6 +102,10 @@ respawn_period=seconds
 This works in conjunction with respawn_max and respawn_delay above to
 decide if a process should not be respawned for some reason.
 
-For example, if respawn_period is 60, respawn_max is 2 and respawn_delay
-is 3 and a process dies more than 4 times, the process will not be
-respawned and the supervisor will terminate.
+For example, if a respawn period is 10, respawn_max is 2 the process
+would need to die 3 times within 10 seconds to no longer be respawned.
```

Comment:
Just a minor change for clarity:

For example, if respawn period is 10 and respawn_max is 2, the process
would need to die 3 times within 10 seconds to no longer be respawned.